atlas-chart/src/main/scala/com/netflix/atlas/chart/util/PngImage.scala

```diff
@@ -87,7 +89,10 @@ object PngImage {
 
   def userError(imgText: String, width: Int, height: Int): PngImage = {
     val userErrorYellow = new Color(0xFF, 0xCF, 0x00)
-    error(imgText, width, height, "USER ERROR:", Color.BLACK, userErrorYellow)
+    val clampedWidth = math.min(width, GraphConstants.MaxWidth)
+    val clampedHeight = math.min(height, GraphConstants.MaxHeight)
```

Comment:
This will be done when calling error, so shouldn't need to be repeated here

Reply:
Ok I will remove this